hydra/_internal/core_plugins/bash_completion.py

```diff
@@ -33,7 +33,15 @@ def install(self) -> None:
         true
     fi
 
-    if ! [ -x "$(command -v $helper)" ]; then
+    EXECUTABLE=($(command -v $helper))
+    if [ "$HYDRA_COMP_DEBUG" == "1" ]; then
+        printf "\\nhelper='$helper'\\n"
+        printf "EXECUTABLE='$EXECUTABLE'\\n"
+        printf "EXECUTABLE_FIRST='${EXECUTABLE[0]}'\\n"
+        printf "EXECUTABLE AS ARRAY:\\n"
+        printf "\\t%s\\n" ${EXECUTABLE[@]}
+    fi
```

Comment:
I mean the whole thing. I don't think it's worth destroying the debug output for this.
```suggestion
```

Reply:
I use debug output (EXECUTABLE_FIRST line) during my test. And all of these lines were useful for me during debugging. Packaging them in one line will destroy clarity of those lines. So I hesitate to package them all into one line.

Reply:
Your debugging is now over. This will get in the way of anyone else that wants to debug the completion.

I am not actually seeing a test here, a comment with code does not count :)